app/i18n/locales/ja-JP.json

```diff
@@ -377,36 +377,5 @@
   "wallet.transaction.type": "{currency} 取引",
   "wallet.transaction.type.exchange": "交換",
   "wallet.transaction.type.intrawallet": "{currency} ウォレット内トランザクション",
-  "wallet.transaction.type.multiparty": "{currency} 複数人数トランザクション",
-  "wallet.trezor.dialog.common.step.link.helpYoroiWithTrezor": "https://youtu.be/Dp0wXwtToX0",
-  "wallet.trezor.dialog.common.step.link.helpYoroiWithTrezor.text": "Trezorを使ったヨロイの使用方法はこちら",
-  "wallet.trezor.dialog.connect.button.label": "接続",
-  "wallet.trezor.dialog.next.button.label": "次へ",
-  "wallet.trezor.dialog.save.button.label": "保存",
-  "wallet.trezor.dialog.step.about.introText.line.1": "ハードウェアウォレットは小さなUSB機器で、ウォレットのセキュリティを高めます。",
-  "wallet.trezor.dialog.step.about.introText.line.2": "秘密鍵がハードウェアウォレットに残らないので、さらに高い安全性を担保します。",
-  "wallet.trezor.dialog.step.about.introText.line.3": "お使いのコンピューターがマルウェアやフィッシング詐欺などにより損なわれたとしても、資金は守られます。",
-  "wallet.trezor.dialog.step.about.label": "概要",
-  "wallet.trezor.dialog.step.about.prerequisite.1.part1": " ",
-  "wallet.trezor.dialog.step.about.prerequisite.1.part2.link": "https://github.com/trezor/trezor-core/blob/be58549fd9fe53df237a7318f754b5ec23975425/ChangeLog#L12",
-  "wallet.trezor.dialog.step.about.prerequisite.1.part2.link.text": "Trezor Model T のバージョン2.0.8",
-  "wallet.trezor.dialog.step.about.prerequisite.1.part3": "以上対応。",
-  "wallet.trezor.dialog.step.about.prerequisite.2": "あらかじめTrezorの初期設定を行ってください。",
-  "wallet.trezor.dialog.step.about.prerequisite.3": "Trezorデバイスの画面をロック解除してください。",
-  "wallet.trezor.dialog.step.about.prerequisite.4": "プロセスの実行中は、コンピューターをインターネットに接続したままにしてください。",
-  "wallet.trezor.dialog.step.about.prerequisite.5": "プロセスの実行中は、Trezorデバイスをインターネットに接続したままにしてください。",
-  "wallet.trezor.dialog.step.about.prerequisite.header": "前提条件",
-  "wallet.trezor.dialog.step.connect.introText.line.1": "Trezorデバイスをお使いのコンピューターに接続してから「接続」ボタンを押してください。",
-  "wallet.trezor.dialog.step.connect.introText.line.2": "新しいタブが開かれます。表示される指示に従ってください。",
-  "wallet.trezor.dialog.step.connect.introText.line.3": "このプロセスにより、CARDANOの秘密鍵がヨロイに共有されます。",
-  "wallet.trezor.dialog.step.connect.label": "接続",
-  "wallet.trezor.dialog.step.save.error.101": "保存に失敗しました。インターネット接続を確認して再試行してください。",
-  "wallet.trezor.dialog.step.save.label": "保存",
-  "wallet.trezor.dialog.step.save.walletName.hint": "ウォレット名を入力",
-  "wallet.trezor.dialog.step.save.walletName.info": "Trezorデバイス名を読み出しました。このまま使用するか、任意の名前を入力してください。",
-  "wallet.trezor.dialog.step.save.walletName.label": "ウォレット名",
-  "wallet.trezor.dialog.title.label": "Trezorハードウェアウォレットに接続",
-  "wallet.trezor.error.101": "Trezor.ioへの接続に失敗しました。インターネット接続を確認して再試行してください。",
-  "wallet.trezor.error.102": "アクセス許可が得られませんでした。再試行してください。",
-  "wallet.trezor.error.103": "キャンセルされました。再試行してください。"
```

Comment:
Why this big chunk is just removed? O_o

Reply:
its renamed from `wallet.trezor.dialog` -> `wallet.connect.trezor.dialog`

Reply:
Understood, thank you!